wallet/src/nightfall-browser/services/finalise-withdrawal.js

```diff
@@ -40,18 +40,15 @@ export async function finaliseWithdrawal(transactionHash, shieldContractAddress)
     SHIELD_CONTRACT_NAME,
     shieldContractAddress,
   );
-  try {
```

Comment:
Same here, we're retiring the try-catch, can we leave it there?

Reply:
Yes, it wasn't doing anything.